playground/app.vue

```
@@ -3,65 +3,70 @@
-    (errors) => {
-      // eslint-disable-next-line no-console
-      console.log('Nope!', errors)
-    }
-  )
+  // handleSubmit wraps the user callback: validation failure auto-populates
+  // signupForm.errors, success clears it. The user's onSubmit can then parse
+  // a server response via parseApiErrors() and write the result with
+  // setFieldErrors(...) to layer server-side errors on top.
+  const onSubmit = signupForm.handleSubmit(async (values) => {
+    // simulate server returning a 422
+    const apiResult = parseApiErrors(
```

ESLint

[error] 'parseApiErrors' is not defined.

(no-undef)